sdk/servicebus/service-bus/src/core/messageReceiver.ts

```diff
@@ -999,32 +999,30 @@ export class MessageReceiver extends LinkEntity {
     operation: DispositionType,
     options?: DispositionOptions
   ): Promise<any> {
-    return new Promise((resolve, reject) => {
+    const settleMessagePromise = new Promise((resolve, reject) => {
       if (!options) options = {};
       if (operation.match(/^(complete|abandon|defer|deadletter)$/) == null) {
         return reject(new Error(`operation: '${operation}' is not a valid operation.`));
       }
-      this._clearMessageLockRenewTimer(message.messageId as string);
       const delivery = message.delivery;
       const timer = setTimeout(() => {
-        this._deliveryDispositionMap.delete(delivery.id);
-
         log.receiver(
           "[%s] Disposition for delivery id: %d, did not complete in %d milliseconds. " +
             "Hence rejecting the promise with timeout error.",
           this._context.namespace.connectionId,
           delivery.id,
-          Constants.defaultOperationTimeoutInSeconds * 1000
+          10 * 1000
```

Comment:
Reason to remove `Constants.defaultOperationTimeoutInSeconds`?
Same a few lines below.